python/tvm/micro/contrib/zephyr.py

```diff
@@ -630,6 +632,8 @@ def __init__(self, base_dir, startup_timeout_sec=5.0, qemu_debugger=None, **kwar
         self.fd_transport = None
         self.pipe_dir = None
         self.qemu_debugger = qemu_debugger
+        self._queue = queue.Queue()
+        self._qemu_ready_msg = "Qready"
```

Comment:
if you leave this in here, promote it to a constant and add a comment explaining its function